src/wowman/core.clj

```diff
@@ -41,9 +41,11 @@
 
 (def colours (utils/nav-map-fn -colour-map))
 
-(defn paths
-  "returns a map of paths whose location may vary depending on the location of the current working directory"
-  [& path]
+(defn generate-path-map
+  "generates a map of filesystem paths whose location may vary according to the current working directory and environment variables.
```

Comment:
"generates filesystem paths whose location may vary based on the current working directory and environment variables."